### ISSUE TITLE
Get Involved Page: email addresses do not have mailto links

### ISSUE THREAD

**Issue by kanarinka** [engagementgamelab/boston-civic-media#73]
Links on the Get Involved page (email addresses) aren't hyperlinks

**Comment by jpailthorpe** [comment]
#49 was supposed to be address these links, but also Aidan and I flagged this content as needing to be rewritten to make more sense contextually weeks ago and got push back. see #32

**Comment by kanarinka** [comment]
Can we either hyperlink those email addresses or take them out please?

**Comment by jpailthorpe** [comment]
If removing is fine, I can do that today. If however it is important to link, we will just need to add this in the model which is something Erica will need to do. @kanarinka once you let us know which you prefer (keep the info and make sure the mailto links are in place, or remove the contacts alltogether) I will re-assign the issue appropriately. 

fwiw I think it is worth revisiting #32 before tackling this, but happy to remove emails now if its important to do that.

**Comment by kanarinka** [comment]
I'm happy with removing email addresses or creating mailto links, whatever is easier and takes less time.

**Comment by esalling23** [comment]
@kanarinka We can do this, however it opens up our websites for email-stealing bots that will send spam. Generally in our websites we have 'xxx at emerson.edu' as the format to avoid this.

**Comment by kanarinka** [comment]
Agreed but spam filters have gotten way better. If we put an email address
on the website it should be a link IMO. Let's give a shot at mailto s and
if the ppl involved are inundated with spam we can backtrack.

**Comment by esalling23** [comment]
All emails that come from specific email fields have mail-to links, and any other email links in the enrollment information, for example, should use this syntax in the keystone markdown field:
`[write to me](mailto://my.address@gmail.com)`

**Comment by rebeccamichelson** [comment]
@esalling23 can you please address this for the Get Involved page.

**Comment by esalling23** [comment]
@codeworthy ready for review at qa.bostoncivic.media
